modules/extension/ysld/src/test/java/org/geotools/ysld/parse/UtilTest.java

```diff
@@ -29,12 +27,12 @@
     public void testColor() {
         Factory factory = new Factory();
 
-        assertEquals(new Color(0xFF0000), ((Literal) Util.color("FF0000", factory)).getValue());
-        assertEquals(new Color(0x00FF00), ((Literal) Util.color("00ff00", factory)).getValue());
-        assertEquals(new Color(0x0000FF), ((Literal) Util.color("#0000FF", factory)).getValue());
-        assertEquals(new Color(0xFFFF00), ((Literal) Util.color("0xFFFF00", factory)).getValue());
-        assertEquals(new Color(0x00FFFF),
+        assertEquals("#FF0000", ((Literal) Util.color("FF0000", factory)).getValue());
+        assertEquals("#00FF00", ((Literal) Util.color("00ff00", factory)).getValue());
+        assertEquals("#0000FF", ((Literal) Util.color("#0000FF", factory)).getValue());
+        assertEquals("#FFFF00", ((Literal) Util.color("0xFFFF00", factory)).getValue());
+        assertEquals("#00FFFF",
                 ((Literal) Util.color("rgb(0, 255, 255)", factory)).getValue());
-        assertEquals(new Color(0xFF00FF), ((Literal) Util.color("f0f", factory)).getValue());
+        assertEquals("#FF00FF", ((Literal) Util.color("f0f", factory)).getValue());
```

Comment:
These tests should still work as-is. If they don't, you may have broken some other behaviour in YSLD.